src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -239,22 +266,39 @@ private static void updateAuthorList(RepoConfiguration config) throws NoAuthorsW
         }
     }
 
-    private static void generateIndividualRepoReport(
-            CommitContributionSummary commitSummary, AuthorshipSummary authorshipSummary, String repoReportDirectory) {
-        CommitReportJson commitReportJson = new CommitReportJson(commitSummary, authorshipSummary);
-        FileUtil.writeJsonFile(commitReportJson, getIndividualCommitsPath(repoReportDirectory));
-        FileUtil.writeJsonFile(authorshipSummary.getFileResults(), getIndividualAuthorshipPath(repoReportDirectory));
+    /**
+     * Adds {@code configs} that were not successfully cloned into the list of errors in the
+     * summary report and removes them from the list of {@code configs}.
+     */
+    private static void handleCloningFailed(List<RepoConfiguration> configs,
+            Set<RepoLocation> successfullyClonedRepoLocations) {
+        Iterator<RepoConfiguration> itr = configs.iterator();
+        while (itr.hasNext()) {
+            RepoConfiguration config = itr.next();
+            if (!successfullyClonedRepoLocations.contains(config.getLocation())) {
+                ErrorSummary.getInstance().addErrorMessage(config.getDisplayName(),
+                        String.format(LOG_ERROR_CLONING, config.getLocation()));
+                itr.remove();
+            }
+        }
     }
 
     /**
-    * Generates a report at the {@code repoReportDirectory}.
-    */
+     * Generates a report at the {@code repoReportDirectory}.
+     */
     public static void generateEmptyRepoReport(String repoReportDirectory, String displayName) {
         CommitReportJson emptyCommitReportJson = new CommitReportJson(displayName);
         FileUtil.writeJsonFile(emptyCommitReportJson, getIndividualCommitsPath(repoReportDirectory));
         FileUtil.writeJsonFile(Collections.emptyList(), getIndividualAuthorshipPath(repoReportDirectory));
     }
 
+    private static void generateIndividualRepoReport(
+            CommitContributionSummary commitSummary, AuthorshipSummary authorshipSummary, String repoReportDirectory) {
```

Comment:
Try to follow the same parameter ordering as existing apis. Let `String repoReportDirectory` be the first parameter